cypress/support/fragments/orders/orderLines.js

```diff
@@ -205,6 +205,23 @@ export default {
     ]);
   },
 
+  POLineInfodorPhysicalMaterialForRecieve: (orderLineTitleName) => {
+    cy.do([
+      orderLineTitleField.fillIn(orderLineTitleName),
+      orderFormatSelect.choose('Physical resource'),
+      acquisitionMethodButton.click(),
+      SelectionOption('Depository').click(),
```

Comment:
let's move this string to const variables to separate constants file which is located in the support directory

Reply:
Done